index.html

```diff
@@ -1,21 +1,106 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <div class="contener">
+        <div class="text_area_nav">
+        <div class="text_nav1">
+            <div class="scouer"></div>
```

Comment:
Empty div no needed so just removed it please :)

Reply:
I was make this div to create blue  square  in order to the design , if you go to css file you will see the color and other staff :)

Reply:
Lets give it a class name that reflects that please (and maybe a comment to explain this to the Reviewer) `square` or `blue-square` would be fine